src/app/opportunitie/opportunitie.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { GeneralService } from '../services/general/general.service';
+
+@Component({
+  selector: 'app-opportunitie',
+  templateUrl: './opportunitie.component.html',
+  styleUrls: ['./opportunitie.component.scss']
+})
+export class OpportunitieComponent implements OnInit {
+  opportuntie: any[];
+
+  constructor( private readonly generalService: GeneralService,private readonly translate: TranslateService ) {
+   
+   }
+
+  ngOnInit(): void {
+    this.initialize();
+    this.translate.onLangChange.subscribe(_ => {
+      this.initialize();
+    });
+  }
+
+    initialize(){this.opportuntie = [
+      {
+        label: this.generalService.translateString('ID_REAMCAREER'),
```

Comment:
@shreyas1434shinde create array of object with following interface
```{
  label: string,
  img: string,
  eligibilityCriteriaUrl: string,
  applyUrl: string
}```

Reply:
@itsvick its already in array of object format